pkg/cli/values/options.go

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright The Helm Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package values
+
+import (
+	"io/ioutil"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/pkg/errors"
+	"sigs.k8s.io/yaml"
+
+	"helm.sh/helm/pkg/cli"
+	"helm.sh/helm/pkg/getter"
+	"helm.sh/helm/pkg/strvals"
+)
+
+type Options struct {
+	ValueFiles   []string
+	StringValues []string
+	Values       []string
+}
+
+// MergeValues merges values from files specified via -f/--values and
+// directly via --set or --set-string, marshaling them to YAML
+func (opts *Options) MergeValues(settings cli.EnvSettings) (map[string]interface{}, error) {
```

Comment:
Did the tests for `MergeValues` get dropped when moving from one package to another?

Reply:
Possibly. In my initial run through the changes, I noticed that only `mergeValues` (which is unused) was actually being tested in `TestMergeValues`.

Looking closer, I see that `mergeMaps`, which _is_ used in `MergeValues` has the same function signature, and appears to do the same thing, though in a slightly different way?

@adamreese Here's the commit where both `mergeMaps` and `mergeValues` were added: https://github.com/helm/helm/commit/0338576fc5202d003e346f41069d50e4f006a285

Is there any reason we should keep both or choose one over the other?

Since `mergeMaps` is what's currently being used in `MergeValues`, I'll plan to re-add `TestMergeValues` and update it to use `mergeMaps` unless I'm told otherwise.

Reply:
Done. Re-added `TestMergeValues`, testing `mergeMaps`.